src/components/header/controlOption.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function ControlOption(props){
+
+    const { item } = props;
```

Comment:
Can also do it with nested object destructuring, though not required but just a suggestion